src/layers/domain/core/ods_organisation/v3.py

```diff
@@ -15,8 +13,8 @@ class OdsOrganisation(AggregateRoot):
 
     ods_code: str = Field(regex=ODS_CODE_REGEX)
 
-    def create_product_team(self, id: UUID, name: str) -> ProductTeam:
-        product_team = ProductTeam(id=id, name=name, ods_code=self.ods_code)
+    def create_product_team(self, name: str = "", keys: list = []) -> ProductTeam:
```

Comment:
2 comments on the defaults:

* I don't think we should allow default `name` values
* For default lists you need to do `keys: list = None` and then in the function set the default (for info [see this](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument))

so suggestion would be

```suggestion
    def create_product_team(self, name: str, keys: list = None) -> ProductTeam:
        keys = keys or []
```